script/core

```diff
@@ -16,34 +16,34 @@ if [ -z $(which hass) ]; then
     git+git://github.com/home-assistant/home-assistant.git@dev
 fi
 
-if [ ! -d "/workspaces/frontend/config" ]; then
+if [ ! -d "${PWD}/config" ]; then
   echo "Creating default configuration."
-  mkdir -p "/workspaces/frontend/config";
+  mkdir -p "${PWD}/config";
   hass --script ensure_config -c config
   echo "demo:
 
 logger:
   default: info
   logs:
     homeassistant.components.frontend: debug
-" >> /workspaces/frontend/config/configuration.yaml
+" >> "${PWD}/config/configuration.yaml"
 
   if [ ! -z "${HASSIO}" ]; then
   echo "
 # frontend:
-#   development_repo: /workspaces/frontend
+#   development_repo: ${PWD}
 
 hassio:
-  development_repo: /workspaces/frontend" >> /workspaces/frontend/config/configuration.yaml
+  development_repo: ${PWD}" >> "${PWD}/config/configuration.yaml"
   else
   echo "
 frontend:
-  development_repo: /workspaces/frontend
+  development_repo: ${PWD}
 
 # hassio:
-#   development_repo: /workspaces/frontend" >> /workspaces/frontend/config/configuration.yaml
+#   development_repo: ${PWD}" >> "${PWD}/config/configuration.yaml"
   fi
 
 fi
 
-hass -c /workspaces/frontend/config
+hass -c "${PWD}/config"
```

Comment:
While this might solve your case, it breaks mine :smile: 
Using PWD means that you have to execute the script from the root of the repository.

Since this is only intended for devcontainer, let's add this to `.devcontainer/devcontainer.json`
```
  "containerEnv": {
    "WORKSPACE_DIRECTORY": "${containerWorkspaceFolder}"
  },
```

And then use `"${WORKSPACE_DIRECTORY:=/workspaces/frontend/}"` in this file, so folks that are using it right now does not need to rebuild

Reply:
Sounds great 👍